### PR TITLE
Adding opentelemetry-instrumentation, opentelemetry-instrumentation-aiohttp-client

### PR DESCRIPTION
**Pull request by twoodwark**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

@conda-forge/help-python

@mariusvniekerk hi, adding you here.

Note: opentelemetry libs have never had a stable release, so we are on beta versions in main.

### CODE REVIEW

**Comment by twoodwark** [comment]
ping: @mariusvniekerk I'm adding you as maintainer to two packages here.  Is that okay?

**Comment by twoodwark** [comment]
Ping: @conda-forge/help-python can you look at this?

File: recipes/opentelemetry-instrumentation-aiohttp-client/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - aiohttp >=3.0.0,<4.0.0
+    - opentelemetry-api ==0.17b0
```
Comment (mariusvniekerk) [inline recipes/opentelemetry-instrumentation-aiohttp-client/meta.yaml:24]: Should we make use of the jinja version bits here?

Comment (twoodwark) [reply recipes/opentelemetry-instrumentation-aiohttp-client/meta.yaml]: 👍

**Pull request merged**